core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -211,7 +229,32 @@ var TopicsAndSkillsDashboardPage = function() {
       confirmSkillCreationButton,
       'Create skill button takes too long to be clickable');
     confirmSkillCreationButton.click();
-    waitFor.pageToFullyLoad();
+
+    waitFor.newTabToBeCreated('Creating skill takes too long');
+    var allHandles = [];
+    await browser.getAllWindowHandles().then(function(handles) {
```

Comment:
Here, I am iterating through all handles, matching the title and if it matches, doing actions on that tab.